internal/plugin/service/service/schema.go

```diff
@@ -0,0 +1,204 @@
+			},
+			"disk_space_step": schema.StringAttribute{
+				Description: "The default disk space step of the service, possible values depend on the service type, the cloud provider and the project. `disk_space` needs to increment from `disk_space_default` by increments of this size.",
+				Computed:    true,
+			},
+			"disk_space_cap": schema.StringAttribute{
+				Description: "The maximum disk space of the service, possible values depend on the service type, the cloud provider and the project.",
+				Computed:    true,
+			},
+			"service_uri": schema.StringAttribute{
+				Description: "URI for connecting to the service. Service specific info is under \"kafka\", \"pg\", etc.",
+				Computed:    true,
+				Sensitive:   true,
+			},
+			"service_host": schema.StringAttribute{
+				Description: "The hostname of the service.",
+				Computed:    true,
+			},
+			"service_port": schema.Int64Attribute{
+				Description: "The port of the service",
+				Computed:    true,
+			},
+			"service_password": schema.StringAttribute{
+				Description: "Password used for connecting to the service, if applicable",
+				Computed:    true,
+				Sensitive:   true,
+			},
+			"service_username": schema.StringAttribute{
+				Description: "Username used for connecting to the service, if applicable",
+				Computed:    true,
+			},
+			"state": schema.StringAttribute{
+				Description: "Service state. One of `POWEROFF`, `REBALANCING`, `REBUILDING` or `RUNNING`",
+				Computed:    true,
+			},
+			"static_ips": schema.SetAttribute{
+				Description: "Static IPs that are going to be associated with this service. Please assign a value using the 'toset' function. Once a static ip resource is in the 'assigned' state it cannot be unbound from the node again",
+				ElementType: types.StringType,
+				Optional:    true,
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"components": schema.SetNestedBlock{
+				Description: "Service component information objects",
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"component": schema.StringAttribute{
+							Description: "Service component name",
+							Computed:    true,
+						},
+						"host": schema.StringAttribute{
+							Description: "DNS name for connecting to the service component",
+							Computed:    true,
+						},
+						"kafka_authentication_method": schema.StringAttribute{
+							Description: "Kafka authentication method. This is a value specific to the 'kafka' service component",
+							Computed:    true,
+						},
+						"port": schema.Int64Attribute{
+							Description: "Port number for connecting to the service component",
+							Computed:    true,
+						},
+						"route": schema.StringAttribute{
+							Description: "Network access route",
+							Computed:    true,
+						},
+						"ssl": schema.BoolAttribute{
+							Description: "Whether the endpoint is encrypted or accepts plaintext. By default endpoints are " +
```

Comment:
rephrase "service components they may disable encryption", maybe something like "service components that might have encryption disabled"